docs/apache-airflow/security/api.rst

```diff
@@ -22,12 +22,12 @@ API Authentication
 ------------------
 
 Authentication for the API is handled separately to the Web Authentication. The default is to
-deny all requests:
+check the user session:
 
 .. code-block:: ini
 
     [api]
-    auth_backends = airflow.api.auth.backend.deny_all
+    auth_backends = airflow.api.auth.backend.session
```

Comment:
Should we add a note to the versionchanged 2.3 section below that it now uses `session` vs `deny_all`?